pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/LocalPolicies.java

```diff
@@ -19,37 +19,34 @@
 package org.apache.pulsar.common.policies.data;
 
 import static org.apache.pulsar.common.policies.data.Policies.defaultBundle;
-import com.google.common.base.Objects;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
 
 /**
  * Local policies.
  */
+@ToString
+@EqualsAndHashCode
 public class LocalPolicies {
```

Comment:
can this change impact compatibility with existing clusters ?

Jackson Mapper uses Java reflection
How do we guarantee that the JSON code of this version is compatible with the one of the previous version ?

Can we add tests that try to deserialise a JSON produced with the old version ? or do we already have some ?

Reply:
yes, you are right. i'll add a test for this case.

Reply:
@eolivelli add a test case in `LocalPolicesTest.testMakeLocalPoliciesImmutableSerializationCompatibility`. can you take a look again ?